curriculum/templates/curriculum/lessonplan_detail.html

```diff
@@ -14,142 +14,132 @@
-                        {% for resource in lessonplan.resources.all %}
-                            {% spaceless %}
-                                <p>
-                                    <a href="{% url 'detail-lesson-resource' pk=resource.id %}" download="{{ resource.file.name }}" target="_blank">{{ resource.name }}</a>
-                                    <span class="ml-2"> ({{ resource.get_semantic_type_display }})</span>
-                                </p>
-                            {% endspaceless %}
-                        {% endfor %}
-                        {% for resource in lessonplan.resource_links.all %}
-                            {% spaceless %}
-                                <p>
-                                    <a href="{{ resource.link }}" target="_blank">{{ resource.name }}</a>
-                                    <span class="ml-2"> ({{ resource.get_semantic_type_display }})</span>
-                                </p>
-                            {% endspaceless %}
-                        {% endfor %}
-                    {% else %}
-                        <span>There are no files associated with this lesson plan</span>
+                <p class="mb-4 stars text-primary" data-average-rating="{{ average_rating }}" >
+                    <i class="fa-star" aria-hidden="true"></i>
+                    <i class="fa-star" aria-hidden="true"></i>
+                    <i class="fa-star" aria-hidden="true"></i>
+                    <i class="fa-star" aria-hidden="true"></i>
+                    <i class="fa-star" aria-hidden="true"></i>
+                </p>
+                <div class="col-md-8 pl-0">
+                    <p>{{ lessonplan.summary }}</p>
+                </div>
+                <div>
+                    <span>
+                        <span class="sr-only">In-class timeline: </span>
+                        <span class="badge badge-outlined badge-secondary lessonplan-info">{% number_to_words lessonplan.num_classes True %} {% format_duration lessonplan.single_class_time True %} class{{lessonplan.num_classes|pluralize:'es'}}</span>
+                    </span>
+                    <span>
+                        <span class="sr-only">Preparation time: </span>
+                        <span class="badge badge-outlined badge-secondary lessonplan-info">{% format_duration lessonplan.total_prep_time %} total prep time</span>
+                    </span>
+                    <span>
+                        <span class="sr-only">Works best for </span>
+                        <span class="badge badge-secondary badge-outlined lessonplan-info">{{ lessonplan.get_grade_level_display }}</span>
+                    </span>
+                    {% if lessonplan.web_only %}
+                    <span>
+                        <span class="sr-only">Chromebook compatible</span>
+                        <span>Chromebook compatible</span>
```

Comment:
Did you mean for the Chromebook compatible to be it's own badge?

Reply:
Oh ha. I definitely didn't mean for it to look like that, but now I realize it's not specified in the wireframes. I think I will make it its own badge for now and @cyan478 can make that decision when she gets the chance.